interfaces/cython/cantera/test/test_transport.py

```diff
@@ -88,6 +88,17 @@ def test_add_species_multi(self):
         self.assertNear(gas1.thermal_conductivity, gas2.thermal_conductivity)
         self.assertArrayNear(gas1.multi_diff_coeffs, gas2.multi_diff_coeffs)
 
+    def test_species_visosities(self):
+
+        state = 500, 2e5, 'H2:0.3, O2:0.28, CH4:0.02, H2O:0.3, N2:0.1'
+        gas1 = ct.Solution(thermo='IdealGas', species=[
+            s for s in ct.Species.listFromFile('gri30.xml')])
```

Comment:
Is there a reason for instantiating this way, rather than just `ct.Solution('gri30.xml')`?

Reply:
Nope, I just copied from `test_add_species_multi` above. I'll just use `self.phase` from the `setUp` function instead